### DISCUSSION TITLE
What does .nv stand for?

### DISCUSSION THREAD

**Discussion by Tevemadar** [softmatterlab/BRAPH-2-Matlab#1754] [General]
I do see what it is, number of vertices, coordinate triplets of them, then number of triangles, and triplets of 1-based vertex indices. But where does this format comes from? .nv is such a short sequence of characters that attempts with popular search engines just do not help.

The background of this question is that our group develops the "Waxholm Space Atlas of the Sprague Dawley Rat Brain", and a user of BRAPH-2 asked us to provide them "the .nv files for the surface of the atlas", which request didn't mean anything to us at the beginning, but finally they told us it's needed for BRAPH-2. I can fulfil their request, internally we use a similar binary format in one of our tools, and modifying the converter to create a text file instead is not a problem. But it still nags me what the actual name of this format could be.

**Comment by iok-ui** [comment]
Hi,

Thank you for reaching out and providing the context.

The .nv format is closely associated with BrainNet Viewer, a widely used tool for visualising human brain networks and cortical surfaces. As you said, it encodes surface geometry, including vertex coordinates and triangular faces, using 1-based indices. Its simplicity makes it a convenient choice for surface visualisation tasks.

In BRAPH-2, the .nv format is employed to facilitate the visualisation of brain surfaces alongside analyses. While BRAPH-2 supports standard neuroimaging formats like NIfTI for volumetric data, .nv provides an efficient way to handle brain surface meshes.

The ability to generate .nv files for your “Waxholm Space Atlas” will allow BRAPH-2 users to incorporate these surfaces into their analyses. Thank you.

I hope this explanation clarifies the purpose and origins of the .nv format. Please don’t hesitate to reach out if you have further questions.

Best regards,
Yu-Wei

**Comment by Tevemadar** [comment]
Thank you very much for the explanation. The user in question has received the surface mesh file in the meantime and reported back that they successfully used it with BRAPH-2, but did not go into details.

Best regards,
Gergely
